src/module/apps/itemImport/importer/VehicleImporter.ts

```diff
@@ -0,0 +1,86 @@
+import { DataImporter } from './DataImporter';
+import { ImportHelper } from '../helper/ImportHelper';
+import { VehicleParser } from '../parser/vehicle/VehicleParser';
+import { Constants } from './Constants';
+import { SR5Actor } from '../../../actor/SR5Actor';
+
+export class VehicleImporter extends DataImporter<Shadowrun.VehicleActorData, Shadowrun.VehicleData> {
+    public files = ["vehicles.xml"];
+
+    CanParse(jsonObject: object): boolean {
+        return jsonObject.hasOwnProperty('vehicles') && jsonObject['vehicles'].hasOwnProperty('vehicle');
+    }
+
+    ExtractTranslation() {
+        if (!DataImporter.jsoni18n) {
+            return;
+        }
+
+        let jsonVehiclei18n = ImportHelper.ExtractDataFileTranslation(DataImporter.jsoni18n, this.files[0]);
+        this.categoryTranslations = ImportHelper.ExtractCategoriesTranslation(jsonVehiclei18n);
+        this.itemTranslations = { ... ImportHelper.ExtractItemTranslation(jsonVehiclei18n, 'vehicles', 'vehicle'),
+                                  ... ImportHelper.ExtractItemTranslation(jsonVehiclei18n, 'weapons', 'weapon') };
+    }
+
+    async createFolders(chummerData: object, translations: Record<string, string>): Promise<Record<string, any>> {
+        const categories = (chummerData as { categories: any })?.categories?.category || [];
+
+        const { drones, vehicles } = categories.reduce(
+            (acc: { drones: any[]; vehicles: any[] }, category: { _TEXT: string }) => {
+                if (category._TEXT?.toLowerCase().startsWith("drones:")) {
+                    const cleanedCategory = { ...category, _TEXT: category._TEXT.replace(/^drones: /i, '').trim() };
+                    acc.drones.push(cleanedCategory);
+                } else {
+                    acc.vehicles.push(category);
+                }
+                return acc;
+            },
+            { drones: [], vehicles: [] }
+        );
+
+        const dronesData = { categories: { category: [...drones] } };
+        const vehiclesData = { categories: { category: [...vehicles] } };
+
+        const droneFolders = await ImportHelper.MakeCategoryFolders("Actor", dronesData, 'Drones', translations);
+        const vehicleFolders = await ImportHelper.MakeCategoryFolders("Actor", vehiclesData, 'Vehicles', translations);
+
+        const folders = { ...droneFolders, ...vehicleFolders };
+        folders['Other'] = await ImportHelper.GetFolderAtPath("Actor", `${Constants.ROOT_IMPORT_FOLDER_NAME}/Vehicles/Other`, true);
+
+        return folders;
+    }
+
+    async Parse(chummerData: object, setIcons: boolean): Promise<StoredDocument<SR5Actor>[]> {
+        let actors: Shadowrun.VehicleActorData[] = [];
```

Comment:
For variables that are never re-assigned I'd advice to use const. This goes for jsonVehiclei18n, actors, jsonDatas, jsonData and actor.
We will also start enforcing this using eslint shortly. You can allready get these as warnings using a eslint extension for whatever IDE/editor you're using. For VSCode that would be 'ESLint' by Microsoft.

I'm aware that these are still all over the place. All these places have already been fixed in another branch by me. :)